metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -572,11 +596,11 @@ object BuildServerConnection {
   /**
    * Establishes a new build server connection with the given input/output streams.
    *
-   * This method is blocking, doesn't return Future[], because if the `initialize` handshake
-   * doesn't complete within a few seconds then something is wrong. We want to fail fast
-   * when initialization is not successful.
+   * This method is blocking, doesn't return Future[], because if the `initialize` handshake doesn't complete within a
```

Comment:
I still haven't worked out the kinks in my auto-formatting, but it looks like the metals format also thinks this is valid, so I can't undo it. I'll try and work on this some more and see why my editor wants to mess with these so much.